src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/SpecimenMapper.java

```diff
@@ -4,11 +4,11 @@
 import org.springframework.stereotype.Component;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
 
-import java.util.Optional;
+import java.util.List;
 
 @Component
 public class SpecimenMapper {
-    public Optional<Specimen> map(final Message message) {
-        return Optional.empty();
+    public List<Specimen> mapToSpecimens(final Message message) {
```

Comment:
If for any reason these segments don't exist, would we return some kind of default or throw an exception? Assuming this will be covered in another ticket

Reply:
If we consume the list with streams – without any preamble (I mean, setting things up ready for the main data on the assumption that there is such data) – then an empty list should be fine I guess?

Reply:
Is it safe to always assume there's data? For now, returning defaults is fine i think

Reply:
Yeah this should be updated as part of the Specimen mapper story (https://gpitbjss.atlassian.net/browse/NIAD-975)

Reply:
Ok 👍